### DISCUSSION TITLE
2021-05-17

### DISCUSSION THREAD

**Discussion by jjunyjjuny** [rla36/airbnb#5] [Daily]
# 2021-05-17 하루 회고하기

## 오늘 진행한 일

- 각종 협업 룰 논의
- 같이 기획서 보면서 API 협의

### IOS

-

### Back

-

### Front

-

**Comment by jjunyjjuny** [comment]
오랜만에 솔플 미션이다. 사실 아직 협업하는 근육이 부족해서 혼자하는게 좀 더 편하긴하다.
특히 이번 미션처럼 typescript, UI 라이브러리 등 새로운 기술을 학습하고 적용할 때는 더욱더 그런 듯 하다. 내 스케쥴대로 맞출 수 있으니까. 일단 저번 미션에는 우테캠에 신경쓰느라 미션을 제대로 하지 못해서 완성도가 많이 아쉬웠는데, 이번에는 코쿼에만 집중해서 할 수 있는 최대한을 해보려고 한다. 이제껏 사용하지 않았던 훅도 사용해보고 이후 recoil로 리팩토링 할듯. 
루카스 요구사항은 참고만하고 내 흐름대로 진행할 예정이다.

**Comment by zzisun** [comment]
이번 프로젝트는 그동안 잘하고 싶었던 테스트코드를 최대한 사용해보려고 한다. 기획서에 따라 Task를 나누고 custom calendar를 살펴보았다!

**Comment by rla36** [comment]
이번 미션은 최대한 TDD로 진행하고 싶다. 지금까지는 기능구현에 급급해 테스트 코드를 작성할 시간이 없었는데 이번에는 테스트도 하면 좋겠다. 디비 설계가 걱정이 되는데 그래도 야구 미션보다는 단순할 것 같아서 다행이다. 지난 번에 디비 설계를 너무 급하게 해서 시간이 지날수록 문제가 잔뜩 터져나왔었는데, 이번에는 그런 일이 없도록 처음부터 신중하게 설계해야겠다.
